src/platform/logging/impl/cc13x2_26x2/Logging.cpp

```diff
@@ -133,7 +107,7 @@ extern "C" void cc13x2_26x2Log(const char * msg, ...)
     va_end(v);
 }
 
-#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
```

Comment:
You replaced `CHIP_DEVICE_CONFIG_ENABLE_THREAD` with `CHIP_ENABLE_OPENTHREAD`. Do you think we should get rid of `CHIP_DEVICE_CONFIG_ENABLE_THREAD` completely as a follow up? In case of BLE it makes sense to maintain two variables: `CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE` and `CONFIG_NETWORK_LAYER_BLE` since it is possible to use CHIP BLE stack without the BLE device layer, but I'm not sure if there's a similar use case for OpenThread.

Reply:
The problem I had was scoping: CHIP_DEVICE_CONFIG_* is only avalable in src/platform and trying to include that caused a lot of circular dependencies.

The flags seem equivalent as I cannot see why one would have 'ENABLE_OPENTHREAD' one value and 'DEVICE_CONFIG_ENABLE_OPENTHREAD' different. I would propose to keep only one, however a full scrub should probably be done as a followup as this PR is quite large still.